ui/about_page.go

```diff
@@ -122,5 +131,9 @@ func (pg *aboutPage) layoutRows(gtx layout.Context) layout.Dimensions {
 	})
 }
 
-func (pg *aboutPage) handle()  {}
+func (pg *aboutPage) handle() {
+	if pg.licenseRow.Clicked() {
+		pg.common.changePage(PageLicense)
```

Comment:
Use `changeFragment` instead.
```suggestion
		pg.common.changeFragment(LicensePage(pg.common), PageLicense)
```